scripts/validate

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -ex
 
-if [ -n "${SKIP_VALIDATE}" ]; then
+if [ -z "${SKIP_VALIDATE}" ]; then
```

Comment:
This will cause CI builds to no longer validate.

Reply:
-n doesn't work locally. Got another suggestion?

Reply:
Ah, I think -n treats "" as being "non-zero"

Reply:
So, remove the quotes around the var expansion and we should be good